data-esnext.js

```diff
@@ -484,24 +484,39 @@ exports.tests = [
   name: 'WeakReferences',
   spec: 'https://github.com/tc39/proposal-weakrefs',
   category: STAGE3,
-  significance: 'large',
+  significance: 'medium',
```

Comment:
why the downgrade? this is a fundamentally new capability.

Reply:
No good reason.  I'll change it back.